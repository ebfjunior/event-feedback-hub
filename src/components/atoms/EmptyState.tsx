export function EmptyState({ message = 'No feedback yet.' }: { message?: string }) {
  return (
    <div className="rounded-md border bg-card p-6 text-center text-sm text-muted-foreground">
      {message}
    </div>
  );
}


