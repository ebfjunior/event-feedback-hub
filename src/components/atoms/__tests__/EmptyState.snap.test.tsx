import React from 'react';
import { render } from '@testing-library/react';
import { EmptyState } from '../EmptyState';

describe('EmptyState (snapshot)', () => {
	it('renders with default message', () => {
		const { container } = render(<EmptyState />);
		expect(container.firstChild).toMatchSnapshot();
	});

	it('renders with custom message', () => {
		const { container } = render(<EmptyState message="Nothing here" />);
		expect(container.firstChild).toMatchSnapshot();
	});
});


