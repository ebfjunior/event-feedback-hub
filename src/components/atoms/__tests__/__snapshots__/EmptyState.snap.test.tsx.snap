// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`EmptyState (snapshot) > renders with custom message 1`] = `
<div
  class="rounded-md border bg-card p-6 text-center text-sm text-muted-foreground"
>
  Nothing here
</div>
`;

exports[`EmptyState (snapshot) > renders with default message 1`] = `
<div
  class="rounded-md border bg-card p-6 text-center text-sm text-muted-foreground"
>
  No feedback yet.
</div>
`;
